dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/enrollment/AbstractEnrollmentService.java

```diff
@@ -1231,6 +1233,16 @@ protected ImportOptions updateImportOptions( ImportOptions importOptions )
         return importOptions;
     }
 
+    /**
+     * Sorts enrollments according to enrollment identifier.
+     *
+     * @param events the list of events.
+     */
+    private void sortEnrollmentUpdates( List<Enrollment> enrollments )
+    {
+        enrollments.sort( ( a, b ) -> a.getEnrollment().compareTo( b.getEnrollment() ) );
+    }
```

Comment:
Minor: you can also consider using the enhanced Java 8 `Comparator` interface:
`Comparator.comparing(Enrollment::getEnrollment)`

Reply:
Thanks @luciano-fiandesio . Wasn't aware, good to know.